### ISSUE TITLE
Improvement: set the target for a DDI, retrieve the actuel target for a DDI

### ISSUE THREAD

**Issue by hubertmeier** [Constey/nfon-api#1]
Hi, your implementation of the nfon-API seems nice, very useful, i succeded with this Scripts to add and to remove extensions to a skill. It would be nice if this would be possible to a waiting queue too, but the skill seems to be mostly the same.

What i am missing are two subroutines for:

- Retrieving the target for a specific DDI, like 0049.123.456789-0
- Setting the target for a specific DDI, to a waiting queue, a skill, a group and so on

its cruel to set many different target for morning, afternoon, and that different on some weekdays, so i would call the target-set-routines with a cron jobs. I did not succed to modify your routines to get the API work.

The manual for the API says:

**/api/customers/K0002/dids/0048.22.123456-0/destination**

and

**/api/customers/K0002/dids/0048.22.123456-0/destination**

Thank you

**Comment by Constey** [comment]
It looks like you solved it yourself :-) I've added some more functions today, but not for the ddis yet. 
Feel free to add your functions if you like and submit a pull request.

**Comment by hubertmeier** [comment]
yes, but my programming style is not as clean enough for publishing here, I needed only  a quick and dirty function to route the -0 to a group (central queue, or queues with mailbox) and I call my script per crontab to avoid the nfon time control. And I needed a function to add or delete an extension to a queue, so I modified your skill-functions, replacing skill with queue was mostly enough.

I reduced the number checking from 3 to 2 digits. That was. For sure, I can send you the first part of my changes, but it's absolutely not good enough for your script 🙄
